modules/navigation.js

```diff
@@ -0,0 +1,24 @@
+const booksWindow = document.querySelector('.books-section');
+const addWindow = document.querySelector('.add-books');
+const contactWindow = document.querySelector('.contact-section');
+
+export function booksDisp(evt) {
+  evt.preventDefault();
+  addWindow.classList.remove('toggle');
+  booksWindow.classList.add('toggle');
+  contactWindow.classList.add('toggle');
+}
+
+export function addDisp(evt) {
+  evt.preventDefault();
+  booksWindow.classList.remove('toggle');
+  addWindow.classList.add('toggle');
+  contactWindow.classList.add('toggle');
+}
+
+export function contactDisp(evt) {
+  evt.preventDefault();
+  contactWindow.classList.remove('toggle');
+  booksWindow.classList.add('toggle');
+  addWindow.classList.add('toggle');
+}
```

Comment:
- [ ] Well done refactoring out these functions to a module. However, we are not done yet. We are yet to refractor each function to use ES6 syntax. Please eliminate the usage of the `function` keyword and favor ES6 syntax.